README.md

```diff
@@ -14,34 +14,30 @@ By default, the updated markdown package exports a function which takes a string
 
 ```jsx
 import React from 'react';
-import rdmd from "@readme/markdown";
+import rdmdx from '@readme/mdx';
```

Comment:
We calling this package `mdx` or `rdmdx`?

Reply:
I always thought of `RDMD` as an acronym of `@readme/markdown`, and so `rdmdx` would be the shortened form of `@readme/mdx`?

Reply:
works for me